ssv/spectest/generate/state_comparison/tests_MsgProcessingSpecTest/qbft message processing valid justification.json

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
This file existed before

Reply:
@GalRogozinski 
This file was created under `ssv` directory, so the `qbft/.../qbft message processing valid justification.json` was unchanged. Did you intend to make changes to the `qbft/.../qbft message processing valid justification.json` and not to create this one (`ssv/.../qbft message processing valid justification.json`)?